### PR TITLE
[xml] Update Bulgarian localization

### PR DESCRIPTION
**Pull request by rddim**
* Add Pin tab feature · notepad-plus-plus/notepad-plus-plus@ce58b42
* Add ability to open/copy selected files from Search-results · notepad-plus-plus/notepad-plus-plus@b822480
* Add "Minimize / Close to" option for System tray · notepad-plus-plus/notepad-plus-plus@eeb4753
* Fix selected pathnames in search results localization issue · notepad-plus-plus/notepad-plus-plus@4671826
* Code enhancement for file checking · notepad-plus-plus/notepad-plus-plus@c021c86
* other fixes

### CODE REVIEW

**Comment by NEKolev** [comment]
Не е ли по-подходящо да се използва "минимизиране" от "намаляване"?

File: PowerEditor/installer/nativeLang/bulgarian.xml [hunk 1]
```diff
@@ -1661,10 +1663,10 @@
 			<finder-collapse-all value="Свиване на всички"/>
 			<finder-uncollapse-all value="Разгъване на всички"/>
 			<finder-copy value="Копиране на избраните редове"/>
-			<finder-copy-paths value="Копиране пълния път на файловете"/>
+			<finder-copy-paths value="Копиране на избран(ите) път(ища)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/bulgarian.xml:1666]: Need to change

File: PowerEditor/installer/nativeLang/bulgarian.xml [hunk 2]
```diff
@@ -1661,10 +1663,10 @@
...
 			<finder-select-all value="Избиране на всичко"/>
 			<finder-clear-all value="Изчистване на всичко"/>
-			<finder-open-all value="Отваряне на всички"/>
+			<finder-open-all value="Отваряне на избран(ите) път(ища)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/bulgarian.xml:1669]: Need to change

**Review by donho — CHANGES_REQUESTED** [review]
Change the labels to the correct ones

**Comment by rddim** [comment]
@NEKolev 
Възможно е. Опитвам се да използвам колкото се може по-малко чужди думи...

File: PowerEditor/installer/nativeLang/bulgarian.xml (hunk 1 above)
Comment (rddim) [reply PowerEditor/installer/nativeLang/bulgarian.xml]: Done

File: PowerEditor/installer/nativeLang/bulgarian.xml (hunk 2 above)
Comment (rddim) [reply PowerEditor/installer/nativeLang/bulgarian.xml]: Done

**Review by donho — APPROVED** [review]

**Pull request closed without merging**